### ISSUE TITLE
Leather Working

### ISSUE THREAD

**Issue by Armoured-Monkey** [mc-datapacks/review-tracker#68]
Project Page: https://www.planetminecraft.com/data-pack/leather-working/

The only differences between the 1.16 and 1.17 versions of the pack are the names of cauldrons (if I am remembering correctly 😛 )

**Comment by EstEarth202** [comment]
# Certified Compatibility ❌
# Stamp of Quality 💕
# ⚠━━━━━・Important ・━━━━━⚠
## Global ignoring tag ❌
- [x] `global.ignore`

data\leatherworks\functions\check_for_items_in_water_cauldrons.mcfunction 
- `execute as @e[type=minecraft:item]` `tag=!global.ignore` missing

## Namespace Convention ✔️
🔆 I recommend this a bit.
**I'm not forcing you to do this. You can refuse or choose to ignore this. This thing is just Recommendations from my experience only.**

I think you should do this for the file/folder structure. 

**Issue:** Some packs use the `datapack_name` as their namespace.

**Sub Issue:** If you follow Datapack Advancement with defines 3 advancement nodes: `Root`, `Namespace` and `Datapack`.

And because datapack node **must not** be in `/data/global/advancements/folder`. 

Causing the need to take this file to `/data/<namespace>/advancements/`

It became like this `/data/<datapack_name>/advancements/<datapack_name>.json`

You might ask me if I use my name instead of datapack name.
You might have this kind of problem.
```
Your pack1 "armouredMonkey:load"
Your pack2 "armouredMonkey:load",
```
But that's not a problem if you do this.
`armoredMonkey/functions/leatherworks/`
```
data
 ├ <your_name>
 │    ├ advancements
 │    │   ├ <datapack_name>
 │    │   │   ├ <event>
 │    │   │   │   ╰ <player_hurt_entity>.json
 │    │   │   ╰ <place>
 │    │   │       ╰ <place_block>.json
 │    │   ╰ <datapack_name>.json
 │    ├ functions
 │    │   ├ <datapack_name>
 │    │   │   ├ <folder function groups>
 │    │   │   ├ main.mcfunction
 │    │   │   ├ setup.mcfunction
 │    │   │   ╰ uninstall.function
 │    │   ╰ <database, random & etc.>
 │    ├ loot_tables
 │    │   ╰ <datapack_name>
 │    │       ╰ <loot folder groups item, block>
 │    │           ╰ <...>.json
 │    ├ predicates
 │    │   ╰ <datapack_name>
 │    │       ├ <folder predicate groups>
 │    │       ╰ <...>.json
 │    ├ recipes
 │    │   ╰ <datapack_name>
 │    │       ╰ <...>.json
 │    ╰ tags
 │        ╰  <items>
 │            ╰ <datapack_name>
 │                ╰ <...>.json
 ├ <global>
 │    ╰ advancements
 │        ├ <your_name>.json
 │        ╰ root.json
 ╰ <minecraft>
 ```

### Solution
**Fact:** It's creator name.
```
data
 ├ <namespace> <- It doesn't make sense to set it as datapack_name.
 │    ╰ advancements         
 │        ╰ <datapack_name>.json
 ├ <global>
 │    ╰ advancements
 │        ├ <namespace>.json <- "This is creator_head" * That's why
 │        ╰ root.json
```

**Because:** If creator plan to write more datapacks, it will be very helpful when you work on workspace.

![image](https://user-images.githubusercontent.com/66269767/125143176-0aca7c00-e144-11eb-851c-ea9cab34237f.png)

**Result:** `<creator_name>:<datapack_name>` When there are multiple packs, the name on front is to identify who created it and next it to tell what datapack it is.

# 🎯━━━━━・Other ・━━━━━🎯
## Bug report 🐞
`schedule clear leatherworks:check_for_items_in_filled_cauldrons` 1.17 version incorrect function name

# 📝━━━━━・Note・━━━━━📝
✔️ = Followed
🔆 = Advised to use
❌ = Doesn't follow
❓ =  Doubt / Something missing
☑️ = Not required/Skip
- [ ] = Required -> with ❌ Must do, ✔️ Should do, 🔆 Not force

**Comment by Armoured-Monkey** [comment]
Thank you,

I have added the ignore tag to the selector in data\leatherworks\functions\check_for_items_in_water_cauldrons.mcfunction (1.17) and \check_for_items_in_filled_cauldrons (1.16) and fixed that bug in the 1.17 version (thanks for pointing it out 😄 ). This is in Leather Works Version 1.2.6 for Minecraft 1.16 and 1.17.

Thanks for your recommendation, it looks useful and I might consider converting my data packs over to that format as I can see the benefits.

**Comment by EstEarth202** [comment]
# Certified Compatibility ✔️
# Style Guideline 💕
You can now use the [certified banner](https://mc-datapacks.github.io/en/certified_datapacks/index.html) for this datapack. The link to your datapack will also be posted on the #certified-datapacks channel.
Since you already have both Certified Datapacker and Style Guideline Follower roles, you will not be given any more roles this time.

Final comment: Interesting.
